src/main/tools/linux-sandbox-pid1.cc

```diff
@@ -53,9 +53,83 @@
 #include "src/main/tools/linux-sandbox.h"
 #include "src/main/tools/logging.h"
 #include "src/main/tools/process-tools.h"
+#include "src/main/cpp/util/file_platform.h"
+#include "src/main/cpp/util/file.h"
+#include "src/main/cpp/util/strings.h"
+
+using blaze_util::IsDirectory;
+using blaze_util::ToString;
+using blaze_util::WriteFile;
```

Comment:
Could we replace these imported methods by copying the code over? If I'm missing something, please correct me, but it seems like we're pulling in whole library just to get these three methods. I would prefer to stick to our own versions here and keep it simple:

- `ToString(x)` is just `std::to_string(x)`.
- `IsDirectory(x)` is one `stat` call and a condition.
- A version of `WriteFile` already existed in this file.

Reply:
Have you agreed on this?

Reply:
I agree that using a library instead of reinventing the wheel for complex code is important, but... where is the complex code that we're using from `src/main/tools`? (Honest question, maybe I'm just missing something here!)

Reply:
Looking back at the original code, there were fewer new duplicated functions than I thought at the time. So it's fine.

Reply:
Thanks for checking! 😊